tests/integration/Tribe/Language_Test.php

```diff
@@ -0,0 +1,141 @@
+<?php
+
+namespace Tribe;
+
+use Closure;
+use Codeception\TestCase\WPTestCase;
+use MO;
+use PO;
+use Tribe__Main;
+use Tribe__Events__Main;
+
+class Language_Test extends WPTestCase {
+
+	/**
+	 * @dataProvider load_text_domain_data_provider
+	 * @test
+	 */
+	public function it_should_load_text_domain_based_on_conditions( Closure $setup, string $expected_result, ?string $expected_message = null ): void {
```

Comment:
The $expected_message is not used. You could remove it. 

Or use it and remove the conditional check for success and failure.